CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * larva-patterns - Improve Larva CSS documentation UI
 * larva-tokens - Add spacing to tokens
 * larva-tokens - Add contents of default.json to the base JSONs to indicate they are required
+* larva-tokens - Update colors in wwd tokens and default tokens
```

Comment:
These will be in a different release - you can move this item to beneath the "Unreleased changes" heading

Reply:
ah gotcha! fixing